pages/common/ern.md

```diff
@@ -0,0 +1,36 @@
+# ern
+
+> Electrode Native platform command line client.
+> More information: <https://native.electrode.io/reference/index-6>.
+
+- Create a new `ern` application (`MiniApp`):
+
+`ern create-miniapp {{appName}}`
+
+- Run one or more `MiniApps` in the iOS Runner application:
+
+`ern run-ios`
+
+- Run one or more `MiniApps` in the Android Runner application:
+
+`ern run-android`
+
+- Create an Electrode Native container:
+
+`ern create-container --miniapps {{/absolute/path/to/miniapp_directory}} --platform {{ios|android}}`
```

Comment:
```suggestion
`ern create-container --miniapps {{/path/to/miniapp_directory}} --platform {{ios|android}}`
```

Path is absolute because begins with `/`, there is no need to explicitly add `absolute` word inside placeholder.

Reply:
(We also need to fix this in `crontab` and `duplicity`)